ingest/rules/curate.smk

```diff
@@ -125,6 +125,8 @@ rule subset_curated_metadata_columns:
         metadata_fields=",".join(config["curate"]["metadata_columns"]),
     shell:
         r"""
-        tsv-select -H -f {params.metadata_fields} \
-            {input.metadata} > {output.metadata}
+        csv2tsv --csv-delim $'\t' {input.metadata} \
+          | tsv-select -H -f {params.metadata_fields} \
+          | csvtk fix-quotes --tabs  \
+        > {output.metadata}
```

Comment:
_nit_

Instead of doing all the format fixes for `tsv-select`, this can just use `csvtk cut` 
(e.g. [subset_metadata](https://github.com/nextstrain/pathogen-repo-guide/blob/b4001735fa55448f1426ff90cca93c667ca121ce/ingest/rules/curate.smk#L148-L149) in pathogen-repo-guide)

```suggestion
        csvtk cut -t -f {params.metadata_fields} \
          {input.metadata} > {output.subset_metadata}
```